src/views/AddItem.jsx

```diff
@@ -1,3 +1,93 @@
-export function AddItem() {
-	return <p>Hello from the <code>/add-item</code> page!</p>
+import { addItem } from '../api/firebase';
+import { useState } from 'react';
+
+export function AddItem({ listId }) {
+	// declare itemName, daysUntilNextPurchase, and radioSelect
+	// into separate state variables
+	const [itemName, setItemName] = useState('');
+
+	const [daysUntilNextPurchase, setDaysUntilNextPurchase] = useState(null);
+
+	const [radioSelect, setRadioSelect] = useState('soon');
+
+	// Store values inside itemData and sent over to database
+	// otherwise log an error if request fails
+	const onFormSubmit = (event) => {
+		event.preventDefault();
+		const itemData = {
+			itemName,
+			daysUntilNextPurchase,
+		};
+		try {
+			addItem(listId, itemData);
+			alert(`${itemData.itemName} was added to your list`);
+		} catch (error) {
+			console.log(error);
+		}
+	};
+
```

Comment:
Again love the comment describing `onFormSubmit` and the function was easy for me to read. I also admire the use of a `try-catch statement`. This is something we haven't done on our end yet but I see how this can be useful for error handling. 

I agree with everyone, including the code authors @arcan9 @Bustamor, that it would be nice to see some kind of state change to the form once `addItem` finishes, such as a blank form state for new submissions or maybe a redirect to the current list. But I completely understand that this wasn't part of the criteria and that I too would likely to have been hesitant.